src/guide/component-props.md

```diff
@@ -61,6 +61,7 @@ In the two examples above, we happen to pass string values, but _any_ type of va
 
 ```html
 <!-- Including the prop with no value will imply `true`. -->
+<!-- If you don't set is-published's type to Boolean in Props, it will be a empty string instead of true value. -->
```

Comment:
A few fixes:

```suggestion
<!-- If you don't set is-published's type to Boolean in props, it will be an empty string instead of "true" value. -->
```